### PR TITLE
Added SL.nnls wrapper

### PR DESCRIPTION
**Pull request by ledell**
The [subsemble](https://github.com/ledell/subsemble) and [h2oEnsemble](https://github.com/h2oai/h2o/tree/master/R/ensemble) ensemble learning R packages support `SL.*` algorithm wrappers as metalearners, so this is a wrapper for the non-negative least squares (NNLS) algorithm in the `SL.*` wrapper format.  This performs the same functionality as the `method.NNLS` metalearning function, which is used in the `SuperLearner` function.  This function could also be used as a base learner in the SuperLearner library, but will probably not perform that well as a base learner.

### CODE REVIEW

**Comment by ecpolley** [comment]
Thanks! For the algorithm wrapper, do you think we should make the normalization of the coefficients to sum to 1 as an option and not the default? The function for combining the algorithms in the super learner often has the sum of the initial coefficients close to 1, so the normalization doesn't change the coefficient values much. In the super learner, the `Z` matrix has all the columns on the same scale as the outcome so it is natural to impose the convex combination constraint, but that might not be the case with a prediction wrapper.

File: inst/NEWS [hunk 1]
```diff
@@ -1,11 +1,13 @@
 # News for the SuperLearner package. #
 
+
 ---
 Version: 2.0-18
-Date: 2014-04-24
+Date: 2014-04-25
```
Comment (lendle) [inline inst/NEWS]: Should be 2015

**Comment by lendle** [comment]
If you want the constraint that the coefficients sum to one, I think it would be better to use quadprog. Normalizing the coefficients from NNLS will be suboptimal in general.

**Comment by ledell** [comment]
Yes, these are both good points.  I will update the `SL.nnls` wrapper accordingly.  My point in normalizing the weights was to have parity with the `method.NNLS` function, but I think it does make more sense to remove the normalization for the `SL.*`-based wrapper version.  As Sam mentioned, if we do want to normalize the weights, it's probably better to use a `quadprog`-based wrapper similar to `method.NNLS2`.  For now, I will leave `SL.nnls` as un-normalized and if there is a need later on, I will re-add normalization. Good ideas, thanks!

**Pull request merged**